components/Avatar.css

```diff
@@ -0,0 +1,53 @@
+.component-avatar{
+    margin-top: 2em;
+}
+
+.avatar-wrappe{
+    display: flex;
+     
+    align-items: flex-end;
+}
+
+.avatar{
+    width: 12rem;
+    height: 12rem;
+    margin-right: 10px;
+}
+.avatar-ex-lagre{
+    width: 12rem;
+    height: 12rem;
+}
+.avatar-large{
+    width: 10rem;
+    height: 10rem;
+}
+.avatar-medium{
+    width: 8rem;
+    height: 8rem;
+}
+.avatar-small{
+    width: 6rem;
+    height: 6rem;
+}
+.avatar-ex-small{
```

Comment:
may be you can name it as .avatar-xsmall or .avatar-x-small for to be more understandable may be, you may ignore as per your needs.